charts/apisix/templates/service-gateway.yaml

```diff
@@ -28,6 +28,7 @@ metadata:
     {{- include "apisix.labels" . | nindent 4 }}
 spec:
   type: {{ .Values.gateway.type }}
+  externalTrafficPolicy: {{ .Values.gateway.externalTrafficPolicy }}
```

Comment:
We can set this field only if the service type is `LoadBalancer`.

Reply:
Both `LoadBalancer` and `NodePort` are affected by this parameter.

Reply:
Both `LoadBalancer` and `NodePort` are affected by this parameter. 
I don’t think it is necessary to add judgment here.

Reply:
Got it.